### PR TITLE
PS-387

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
This PR will save the artist name if he edits it.

### CODE REVIEW

File: lib/easel_provider.dart [hunk 1]
```diff
@@ -168,9 +168,27 @@ class EaselProvider extends ChangeNotifier {
     return false;
   }
 
+
+
+   saveArtistName(name) {
+     localDataSource.saveArtistName(name);
+  }
+
+  toCheckSavedArtistName(){
```
Comment (kjawadDeveloper2) [inline lib/easel_provider.dart]: Add method type.

File: lib/easel_provider.dart [hunk 2]
```diff
@@ -168,9 +168,27 @@ class EaselProvider extends ChangeNotifier {
     return false;
   }
 
+
+
+   saveArtistName(name) {
```
Comment (kjawadDeveloper2) [inline lib/easel_provider.dart]: Add method type

File: lib/utils/dependency_injection/dependency_injection_container.dart [hunk 3]
```diff
@@ -5,12 +5,11 @@ import 'package:dio/dio.dart';
 import 'package:easel_flutter/datasources/local_datasource.dart';
 import 'package:easel_flutter/datasources/remote_datasource.dart';
 import 'package:easel_flutter/easel_provider.dart';
-import 'package:easel_flutter/env.dart';
 import 'package:get_it/get_it.dart';
 import 'package:shared_preferences/shared_preferences.dart';
 
 final sl = GetIt.instance;
-
+const apiKey = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiJkaWQ6ZXRocjoweGQ1QmMxZjZDODZEYzFEMmQ2OGI4NDU3YzUyMzM1NTIzM0M5MGYxQjgiLCJpc3MiOiJuZnQtc3RvcmFnZSIsImlhdCI6MTYzNjQ0NzkyMDE2OSwibmFtZSI6IkVhc2VsX2tleXMifQ.8fwxJ2NRDOlpeYLqZfgp6TEzHkwJ15VihLJeuFGP4co";
```
Comment (kjawadDeveloper2) [inline lib/utils/dependency_injection/dependency_injection_container.dart]: Remove this from here.

**Review by kjawadDeveloper2 — COMMENTED** [review]
Kindly solve these minor issues.

**Comment by kjawadDeveloper2** [comment]
Need to rewrite the history because of addition of url key.

File: lib/utils/dependency_injection/dependency_injection_container.dart [hunk 4]
```diff
@@ -5,12 +5,11 @@ import 'package:dio/dio.dart';
 import 'package:easel_flutter/datasources/local_datasource.dart';
 import 'package:easel_flutter/datasources/remote_datasource.dart';
 import 'package:easel_flutter/easel_provider.dart';
-import 'package:easel_flutter/env.dart';
 import 'package:get_it/get_it.dart';
 import 'package:shared_preferences/shared_preferences.dart';
 
 final sl = GetIt.instance;
-
+const apiKey = "***REMOVED***";
```
Comment (kjawadDeveloper2) [inline lib/utils/dependency_injection/dependency_injection_container.dart]: Looks like done.

**Review by mustafapylons — APPROVED** [review]
Looks good to me.

**Review by kiroj305 — APPROVED** [review]

**Pull request merged**